metadata/index.html

```diff
@@ -2177,19 +1936,11 @@
       </p>
     </section>
     <section class="appendix" id="changes">
-      <h2>Changes since the working draft of 08 January 2015</h2>
-      <p>The document has undergone substantial changes since the last working draft. Below are some of the changes made:</p>
+      <h2>Changes since the working draft of 16 April 2015</h2>
```

Comment:
I'll put it back; I considered this, but it meant creating sub-sections in a changes appendix, and strictly speaking, this is describing changes from the previous WD, but there's value in keeping the history in the document, at least through LCCR.